frontend_tests/node_tests/compose.js

```diff
@@ -43,6 +48,70 @@ people.initialize_current_user(me.user_id);
 people.add(alice);
 people.add(bob);
 
+i18n.init({
```

Comment:
You should create a stub version of `i18n`.  See the tests for `settings_org.js` as an example.